docs/source/en/trainer.md

```diff
@@ -108,6 +108,12 @@ You can save your checkpoints (the optimizer state is not saved by default) to t
 
 When you resume training from a checkpoint, the [`Trainer`] tries to keep the Python, NumPy, and PyTorch RNG states the same as they were when the checkpoint was saved. But because PyTorch has various non-deterministic default settings, the RNG states aren't guaranteed to be the same. If you want to enable full determinism, take a look at the [Controlling sources of randomness](https://pytorch.org/docs/stable/notes/randomness#controlling-sources-of-randomness) guide to learn what you can enable to make your training fully deterministic. Keep in mind though that by making certain settings deterministic, training may be slower.
 
+If you encounter an error loading the model when resuming from a checkpoint, it may be caused by a lack of support in the Trainer for loading the type of model in your checkpoint.  If you are able to load the model yourself and pass that as the _model_ parameter to the Trainer constructor, you can use the _without_checkpoint_model_ parameter to the train() method to force the Trainer to skip re-loading your model during resumption from a checkpoint.  e.g.:
```

Comment:
```suggestion
If you encounter an error loading the model when resuming from a checkpoint, it may be caused by a lack of support in the Trainer for loading the type of model in your checkpoint. If you are able to load the model yourself and pass that as the `model` parameter to the Trainer constructor, you can use the `without_checkpoint_model` parameter in the train() method to force the Trainer to skip re-loading your model when resuming from a checkpoint e.g.:
```